tsconfig.json

```diff
@@ -18,7 +18,8 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react"
+    "jsx": "react",
+    "noFallthroughCasesInSwitch": true
```

Comment:
any reason why we are adding this to our ts compiler?

Reply:
no reason! didn't even catch that, will remove